phoenix-spark/src/it/scala/org/apache/phoenix/spark/PhoenixSparkIT.scala

```diff
@@ -16,12 +16,17 @@ package org.apache.phoenix.spark
 import java.sql.DriverManager
 import java.util.Date
 
+import org.apache.phoenix.mapreduce.util.PhoenixConfigurationUtil
 import org.apache.phoenix.schema.types.PVarchar
-import org.apache.phoenix.spark.datasource.v2.PhoenixDataSource
+import org.apache.phoenix.spark.datasource.v2.{PhoenixDataSource, PhoenixTestingDataSource}
+import org.apache.phoenix.spark.datasource.v2.reader.PhoenixTestingInputPartitionReader
+import org.apache.phoenix.spark.datasource.v2.writer.PhoenixTestingDataSourceWriter
 import org.apache.phoenix.util.{ColumnInfo, SchemaUtil}
+import org.apache.spark.SparkException
 import org.apache.spark.sql.types._
```

Comment:
not relates to this PR, does this mean everything? do we need all of them?

Reply:
Yes it is a wildcard import. Might as well remove all such occurrences. Will remove them throughout.